adapters/amqp-vertx/src/main/java/org/eclipse/hono/adapter/amqp/Config.java

```diff
@@ -120,4 +121,14 @@ public ConnectionEventProducer connectionEventProducer() {
         return new LoggingConnectionEventProducer();
     }
 
+    /**
+     * Exposes the health check server as a Spring bean.
+     *
+     * @return the health check server.
+     */
+    @Bean
+    public HealthCheckServer healthCheckServer() {
```

Comment:
how about moving this into `AbstractAdapterConfig`?

Reply:
Good point. Will do.